### DISCUSSION TITLE
Canvas not workingg

### DISCUSSION THREAD

**Discussion by wwyuro** [thatskymod/Sky-CotL-Scripts#146] [Q&A] [answered]
recently updated canvas to the latest version as well as the other mods.

I've gotten this problem before actually where I try to play and it just keeps putting me on canvas's intro screen over and over, this only occurs when I have the dukun langit mod on but with the other mods it seems completely fine.

I've got the latest update on this mod as well, cleared the cache  of canva as well as restarting my device. I'm not sure if it's a problem on my specific device which is a tablet but I doubt it since I've played with this mod before the problem started to occur.

**Comment by thatskymod** [comment, marked as answer]
Hello,

I apologize for the outage. That said, any time the base game is updated, mods also need to be aligned with the new logic. This may take some time.

Consider trying the newly released [libLangit046.so](https://github.com/thatskymod/Sky-CotL-Scripts/commit/21df5cdfb0eb75710bf49820ee1e234e2ad26abd). This version should resolve any outstanding issues found in its predecessors.
